src/zenml/integrations/tekton/orchestrators/tekton_orchestrator.py

```diff
@@ -439,31 +491,44 @@ def _construct_kfp_pipeline() -> None:
                 "and the pipeline will be run immediately."
             )
 
-        logger.info(
-            "Running Tekton pipeline in kubernetes context '%s' and namespace "
-            "'%s'.",
-            self.config.kubernetes_context,
-            self.config.kubernetes_namespace,
-        )
+        kubernetes_context = self.config.kubernetes_context
+        if kubernetes_context:
+            logger.info(
+                "Running Tekton pipeline in kubernetes context '%s' and "
+                "namespace '%s'.",
+                kubernetes_context,
+                self.config.kubernetes_namespace,
+            )
+        elif self.connector:
+            logger.info(
+                "Running Tekton pipeline with Kubernetes credentials from "
+                "connector '%s'.",
+                str(self.connector),
```

Comment:
I think this is the UUID right, maybe it's more helpful if we can print the name instead?